elasticdl/python/common/args.py

```diff
@@ -491,7 +491,6 @@ def add_common_args_between_master_and_worker(parser):
     parser.add_argument(
         "--distribution_strategy",
         type=str,
-        default="ParameterServerStrategy",
```

Comment:
We can keep the default as `ParameterServerStrategy`. If there are other conflicts for default values, change all default to `ParameterServerStrategy`.

Reply:
In `ParameterStrategy`,  `tf.keras.layers.Embedding` will forcibly replaced with `elasticdl.python.elasticdl.layers.Embedding`. This will fail certain test cases. Maybe a better default strategy should be setup.

Reply:
A related issue is created. #1458